CONTRIBUTING.md

```diff
@@ -87,6 +87,14 @@ pnpm pqb t select.test
 pnpm orm t belongsTo.test
 ```
 
+## VSCode Dev Container setup.
+
+For an isolated dev environment that automatically starts postgres on launch for you:
+
+1. `F1` in VScode and star start typing `Dev Containers: Clone Repository in Named Container Volume` to run the commmand.
+1. For the repo, paste `git@github.com:romeerez/orchid-orm.git` if you're using ssh
```

Comment:
Could these be 1, 2, 3? Markdown is supposed to stay human readable, after all. :)

Also, I think all items should either end with a period, or not end with it - but not inconsistently.

Reply:
To me 1. 1. 1. also looks weird

Reply:
I just spent the day writing a proposal for an overhaul of the migration system and it will be a ton of code... If this tiny PR yields this much bike-shedding that doesn't bode well.

Reply:
Sorry, EOL, tabs, and list numbering are minor things, and they shouldn't bother humans but to be enforced automatically by linters.